src/InfoPage.js

```diff
@@ -1,13 +1,47 @@
-import React from 'react';
+import React, { Component } from 'react';
+import './InfoPage.css';
+import { getSelectedMovie } from './apiCalls';
+import Error from './Error';
 
-const InfoPage = ({ selectedMovie }) => {
+class InfoPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedMovie: {},
+      isError: false,
+      errorMessage: ''
+    };
+  };
 
-  return (
-    <>
-   <h2>{selectedMovie.title}</h2>
-   <img src={selectedMovie.backdrop_path} alt='movie backdrop'></img>
-    </>
-  )
-}
+  componentDidMount = () => {
+    getSelectedMovie(this.props.selectedMovieId)
+    .then(data => {
+      this.setState({
+        selectedMovie: data.movie
+      })
+    })
+    .catch(error => {
+      console.log(error);
+      this.setState({
+        isError: true,
+        errorMessage: error
+      })
+    })
+  };
```

Comment:
Love how you separated the apiCalls and are calling them in your `componentDidMount`.  Nice work! 🎉